guides/embedders/huggingface.mdx

```diff
@@ -0,0 +1,87 @@
+---
+title: Semantic Search with Hugging Face Inference Endpoints - Meilisearch documentation
+description: This guide will walk you through the process of setting up Meilisearch with Hugging Face Inference Endpoints to enable semantic search capabilities.
+---
+
+# Semantic search with Hugging Face Inference Endpoints
+
+## Introduction
+
+This guide will walk you through the process of setting up Meilisearch with [Hugging Face Inference Endpoints](https://ui.endpoints.huggingface.co/) embeddings to enable semantic search capabilities. By leveraging Meilisearch's AI features and Hugging Face Inference Endpoints, you can enhance your search experience and retrieve more relevant results.
+
+## Requirements
+
+To follow this guide, you'll need:
+
+- A [Meilisearch Cloud](https://www.meilisearch.com/cloud?utm_campaign=vector-search&utm_source=docs&utm_medium=huggingface-embeddings-guide) project running version 1.10 or above with the Vector store activated.
+- An Hugging Face account with a deployed inference endpoint. You can sign up for a Hugging Face account at [Hugging Face](https://huggingface.co/).
+- The endpoint url and API key of the deployed model on your Hugging Face account.
+- No backend required.
+
+## Setting up Meilisearch
+
+To set up an embedder in Meilisearch, you need to configure it to your settings. You can refer to the [Meilisearch documentation](/reference/api/settings?utm_campaign=vector-search&utm_source=docs&utm_medium=huggingface-embeddings-guide#update-embedder-settings) for more details on updating the embedder settings.
+
+Hugging Face offers a [wide variety of embedding models](https://ui.endpoints.huggingface.co/catalog?task=sentence-embeddings) you can deploy to embed the content of your documents.
+
+Here's an example of embedder settings using [BAAI/bge-small-en-v1.5](https://huggingface.co/BAAI/bge-small-en-v1.5) model:
```

Comment:
Here the link to the hugging face catalog can be interesting to be kept because it list the category of model Meilisearch can work with (sentence-embeddings)